cds_ils/templates/invenio_app_ils_circulation/mail/cds_checkout.html

```diff
@@ -5,23 +5,30 @@
 {% block body_plain %}
 Dear {{ patron.name }},
 
-your loan for "{{ document.full_title }}" <{{ spa_routes.HOST }}{{ spa_routes.PATHS['literature']|format(pid=document.pid) }}> has started.
+Your loan for "{{ document.full_title }}" <{{ spa_routes.HOST }}{{ spa_routes.PATHS['literature']|format(pid=document.pid) }}> has started.
+
+You selected: {{ circulation_delivery_methods[loan.delivery.method] }}
 
 The due date is {{ loan.end_date }}.
 
 If you did not already pick up the book at the Library desk, be aware that:
-- if you requested a pick up at the Library, your book is ready. You can come to pick it up.
-- if you requested an internal mail delivery, your book has been sent. Delivery estimate is 2-3 working days.
+- if you requested a pick up at the Library, your book is ready. You can come to pick it up,
+- if you requested an internal mail delivery, your book has been sent. Delivery estimate is 2-3 working days,
+- if you have not yet chosen one of the above options, please let us know your preference, by replying to this email.
 {% endblock %}
 
 {% block body_html %}
 Dear {{ patron.name }}, <br/><br/>
 
-your loan for <a href="{{ spa_routes.HOST }}{{ spa_routes.PATHS['literature']|format(pid=document.pid) }}">"{{ document.full_title }}"</a> has <b>started</b>. <br/><br/>
+Your loan for <a href="{{ spa_routes.HOST }}{{ spa_routes.PATHS['literature']|format(pid=document.pid) }}">"{{ document.full_title }}"</a> has <b>started</b>. <br/><br/>
 
+<br/><br/>
+You selected: <b>{{ circulation_delivery_methods[loan.delivery.method] }}</b>
+<br/><br/>
 The due date is <b>{{ loan.end_date }}</b>. <br/><br/>
 
 If you did not already pick up the book at the Library desk, be aware that: <br/>
-- if you requested a pick up at the Library, your book is ready. You can come to pick it up.<br/>
-- if you requested an internal mail delivery, your book has been sent. Delivery estimate is 2-3 working days.<br/>
+- if you requested a pick up at the Library, your book is ready. You can come to pick it up,<br/>
+- if you requested an internal mail delivery, your book has been sent. Delivery estimate is 2-3 working days,<br/>
+- if you have not yet chosen one of the above options, please let us know your preference, by replying to this email.<br/>
```

Comment:
Does it mean that `circulation_delivery_methods[loan.delivery.method]` might fail???